source/classification1.md

```diff
@@ -1827,101 +1716,79 @@ Upsampled data with background color indicating the decision of the classifier.
-# formula `Class ~ Area + Smoothness` (instead of `Class ~ .`) in the recipe.
-# You will also notice that we did not call `prep()` on the recipe; this is unnecessary when it is
-# placed in a workflow.
-
-# We will now place these steps in a `workflow` using the `add_recipe` and `add_model` functions, \index{tidymodels!add\_recipe}\index{tidymodels!add\_model}
-# and finally we will use the `fit` function to run the whole workflow on the `unscaled_cancer` data.
-# Note another difference from earlier here: we do not include a formula in the `fit` function. This \index{tidymodels!fit}
-# is again because we included the formula in the recipe, so there is no need to respecify it:
-```
+from sklearn.pipeline import make_pipeline
 
-```{code-cell} ipython3
-knn_fit = make_pipeline(preprocessor, knn_spec).fit(
-    X=unscaled_cancer.loc[:, ["Area", "Smoothness"]], y=unscaled_cancer["Class"]
+knn_fit = make_pipeline(preprocessor, knn).fit(
+    X=unscaled_cancer, 
+    y=unscaled_cancer["Class"]
 )
 
 knn_fit
 ```
 
 As before, the fit object lists the function that trains the model. But now the fit object also includes information about
-the overall workflow, including the standardizing preprocessing step.
+the overall workflow, including the standardization preprocessing step.
 In other words, when we use the `predict` function with the `knn_fit` object to make a prediction for a new
 observation, it will first apply the same preprocessing steps to the new observation. 
 As an example, we will predict the class label of two new observations:
 one with `Area = 500` and `Smoothness = 0.075`, and one with `Area = 1500` and `Smoothness = 0.1`.
 
-```{code-cell} ipython3
-:tags: [remove-cell]
-
-# As before, the fit object lists the function that trains the model as well as the "best" settings
-# for the number of neighbors and weight function (for now, these are just the values we chose
-#  manually when we created `knn_spec` above). But now the fit object also includes information about
-# the overall workflow, including the centering and scaling preprocessing steps.
-# In other words, when we use the `predict` function with the `knn_fit` object to make a prediction for a new
-# observation, it will first apply the same recipe steps to the new observation. 
-# As an example, we will predict the class label of two new observations:
-# one with `Area = 500` and `Smoothness = 0.075`, and one with `Area = 1500` and `Smoothness = 0.1`.
-```
-
 ```{code-cell} ipython3
 new_observation = pd.DataFrame({"Area": [500, 1500], "Smoothness": [0.075, 0.1]})
 prediction = knn_fit.predict(new_observation)
 prediction
 ```
 
-The classifier predicts that the first observation is benign ("B"), while the second is
-malignant ("M"). {numref}`fig:05-workflow-plot-show` visualizes the predictions that this 
+The classifier predicts that the first observation is benign, while the second is
+malignant. {numref}`fig:05-workflow-plot-show` visualizes the predictions that this 
```

Comment:
below uses `numpy.meshgrid`. I would be inclined to replace with `np.meshgrid` (or say "the `meshgrid` function from `numpy`)